src/components/body/body.js

```diff
@@ -133,6 +136,51 @@ var Body = {
     }
   },
 
+  updateCannonScale: function(){
+    this.el.object3D.getWorldScale(this.worldScale);
+    const scale = this.worldScale;
+    const cannonBody = this.body;
+
+    for (var i=0; i<cannonBody.shapes.length; i++){
+      const cannonShape = cannonBody.shapes[i];
+      const shape = cannonShape.component;
+      if (!shape) continue;
+      const shapeType = shape.data.shape;
+      switch (shapeType){
+        case "sphere":
+          cannonShape.radius = shape.data.radius * scale.x;
+          cannonShape.updateBoundingSphereRadius();
+          break;
+        case "box":
+          const cannonHalfExtents = cannonShape.halfExtents;
+          const halfExtents = shape.data.halfExtents;
+          cannonHalfExtents.set(
+            halfExtents.x * scale.x,
+            halfExtents.y * scale.y,
+            halfExtents.z * scale.z
+          );
+          cannonShape.updateConvexPolyhedronRepresentation();
+          break;
+        //case "cylinder":
+          // https://github.com/MozillaReality/cannon.js/commit/b401ccbec3b9996e31d6ad060028065a432fcff7
+          // cannonShape.resize(shape.data.radiusTop * scale.x, shape.data.radiusBottom*scale.x, shape.data.height*scale.y);
+          // break;
+        default:
+          if (!this.didIssueResizeWarning){
+            this.didIssueResizeWarning = true;
+            console.warn("Cannot resize shape of type: ", shapeType);
+          }
+          break;
+      }
+      const cannonOffset = cannonBody.shapeOffsets[i];
+      const offset = shape.data.offset;
+      cannonOffset.set(offset.x*scale.x, offset.y*scale.y, offset.z*scale.z);
+    }
+    cannonBody.updateBoundingRadius();
+    cannonBody.aabbNeedsUpdate = true;
```

Comment:
I think you want to call `updateMassProperties()` here instead, which will automatically recompute AABB as well.